Automattic/newspack-plugin
# Wizards: Update notice-like elements with the Notice component

### All Submissions:






### Changes proposed in this Pull Request:

PR removes the custom CSS/SVG and use the Notice component instead.

Wizards:
* Health Check
* Performance
* Reader Revenue
* Configure Plugins
* Error Modal (via With Wizards)

### How to test the changes in this Pull Request:

1. Switch to this branch.
2. Check the different pages.
3. Text + icon should be using the `Notice` component

### Other information:
